app/src/androidTest/java/mozilla/lockbox/uiTests/AppRoutePresenterTest.kt

```diff
@@ -61,11 +63,16 @@ open class AppRoutePresenterTest {
             back()
         }
         navigator.checkAtFilterList()
+        navigator.back()
     }
 
     @Test
     fun testSettings() {
-        navigator.gotoSettings()
+        navigator.gotoItemList(true)
+        itemList {
+            tapSettings()
+        }
+        settings { exists() }
```

Comment:
I'm really puzzled why you're replacing these `navigator` calls with other `navigator` calls and `Robot` calls?

Reply:
I was having issues going to the item list view. It was not possible to go there from the welcome view unless going manually through all the onboarding screens. With the latest commit here I was able to fix that. and these Robot calls have been removed